### PR TITLE
Added Smartsupp, Smartlook, Luigi’s Box, Targito, Foxentry

### PR DESCRIPTION
**Pull request by machal**
Added few of third party vendors from the Czech republic

### CODE REVIEW

File: data/entities.json5 [hunk 1]
```diff
@@ -11347,5 +11347,53 @@
...
+      "smartsupp.com",
+      "smartsupp-widget-*.cdn77.org",
+      "smartsupp-files-*.cdn77.org"
+    ]
+  },
+  {
+    "name": "Smartlook",
+    "company": "Smartsupp.com",
+    "homepage": "https://www.smartlook.com/",
+    "categories": ["customer-success"],
```
[the remark this answers is not in the document]
Comment (machal) [reply data/entities.json5]: It is customer chat provider. But correct homepage is https://www.smartsupp.com/. I will try to change it.

File: data/entities.json5 [hunk 2]
```diff
@@ -11347,5 +11347,53 @@
...
+  },
+  {
+    "name": "Smartsupp",
+    "company": "Smartsupp.com",
+    "homepage": "https://www.smartsupp.com",
+    "categories": ["customer-success"],
+    "domains": [
+      "smartsuppchat.com",
+      "smartsupp.com",
+      "smartsupp-widget-*.cdn77.org",
```
[the remark this answers is not in the document]
Comment (machal) [reply data/entities.json5]: I have looked into few of my clients sites an it looks they use only one subdomain:

```
smartsupp-widget-161959.c.cdn77.org
```

File: data/entities.json5 (hunk 1 above)
[the remark this answers is not in the document]
Comment (patrickhulce) [reply data/entities.json5]: isn't that the one above this?

File: data/entities.json5 (hunk 2 above)
[the remark this answers is not in the document]
Comment (patrickhulce) [reply data/entities.json5]: great let's just use that for now then

File: data/entities.json5 (hunk 1 above)
[the remark this answers is not in the document]
Comment (machal) [reply data/entities.json5]: Oh, yes, you are right. This one is not chat (which is Smartsupp), but user session video recorder. Both have the same company. So I think analytics provider is more appropriate. Will change it.

File: data/entities.json5 [hunk 3]
```diff
@@ -11347,5 +11347,53 @@
...
+      "*.smartsupp.com",
+      "smartsupp-widget-*.cdn77.org",
+      "smartsupp-files-*.cdn77.org"
+    ]
+  },
+  {
+    "name": "Smartlook",
+    "company": "Smartsupp.com",
+    "homepage": "https://www.smartlook.com/",
+    "categories": ["customer-success"],
```
Comment (machal) [inline data/entities.json5]:
```suggestion
    "categories": ["analytics"],
```

File: data/entities.json5 [hunk 4]
```diff
@@ -11347,5 +11347,53 @@
...
+  {
+    "name": "Smartsupp",
+    "company": "Smartsupp.com",
+    "homepage": "https://www.smartsupp.com",
+    "categories": ["customer-success"],
+    "domains": [
+      "*.smartsuppchat.com",
+      "*.smartsupp.com",
+      "smartsupp-widget-161959.c.cdn77.org",
+      "smartsupp-files-*.cdn77.org"
```
Comment (patrickhulce) [inline data/entities.json5]: is the number the same for this `-files` domain? can't support the wildcard unfortunately :/

if we don't know it, could just leave it in examples for now?

Comment (machal) [reply data/entities.json5]:
```suggestion:-1
```

Comment (machal) [inline data/entities.json5]:
```suggestion
```

Comment (machal) [reply data/entities.json5]: I've changed it in the last commit.

**Pull request merged**